ArtOfIllusion/src/artofillusion/script/ScriptRunner.java

```diff
@@ -56,8 +59,12 @@ else if (language.equals("BeanShell"))
           }
         catch (Exception e)
           {
+            // TODO FIXME NOOOOOO
             e.printStackTrace();
           }
+        // TODO Use a ScriptOutputPanel or something like this
```

Comment:
Also a good spot for a tracking issue so that we can discuss what that should look like overall.

Reply:
Well, my reasoning was each tool script should have its own output and error (as in stdout /  stderr) panel ; but I wanted to have feedback on the PR before submitting such a change. Should this be another PR?

Reply:
Yeah, let's consider this idea out-of-scope for this PR. Smaller bites are generally more efficient to review and integrate when possible.